src/libappimage/libappimage.cpp

```diff
@@ -95,6 +95,7 @@ void appimage_string_list_free(char** list) {
 bool
 appimage_read_file_into_buffer_following_symlinks(const char* appimage_file_path, const char* file_path, char** buffer,
                                                   unsigned long* buf_size) {
+    auto logger = Logger::instance();
```

Comment:
`log` should be a static method, there's no need to call `instance` every time.

Reply:
Static convenience methods were added to ease the logger usage. The users will have no need to call `log` anymore.